src/utils/cart.jsx

```diff
@@ -0,0 +1,31 @@
+const addToCart = (cart, payload) => {
+  return cart.find((product) => product._id === payload._id)
```

Comment:
suggestion file contains the logical part than you can change it to .js

Reply:
Thanks for the good PR review